File: EulerProblem13.rb
def ep13
  total = 0

  File.open('EulerProblem13.txt', 'r') do |file|
    while (line = file.gets)
      total += line.to_i
    end
  end

  return total.to_s[0,10]

end

puts ep13

# CORRECT

File: EulerProblem13.py
def ep13():
  total = 0

  with open('EulerProblem13.txt', 'r') as f:
    for line in f:
      num = int(line)
      total += num

  # for string in ten_digit_strings:
  #   number = int(string)
  #   total += number
  #
  return total

print ep13()

# CORRECT
